pymatgen/io/lobster/outputs.py

```diff
@@ -408,6 +409,109 @@ def icohpcollection(self):
         return self._icohpcollection
 
 
+class NciCobiList:
+    """
+    Class to read NcICOBILIST (multi-center ICOBI) files generated by LOBSTER.
+
+    Attributes:
+        is_spin_polarized (bool): Boolean to indicate if the calculation is spin polarized.
+        NciCobiList (dict): Dict containing the listfile data of the form:
+        {bond: "number_of_atoms": number of atoms involved in the multi-center interaction,
+                "ncicobi": {Spin.up: Nc-ICOBI(Ef) spin up, Spin.down: ...}},
+                "interaction_type": type of the multi-center interaction
+    """
+
+    def __init__(self, filename: str | None = None):
```

Comment:
```suggestion
    def __init__(self, filename: str | None = "NCICOBILIST.lobster"):
```
I think the use case is that someone has computed multicenter COBIs and then wants to read the file. Thus, the default name should be there. In a larger code, I would check if the file existed, before I use the class.

Reply:
I just commented this from my phone as I randomly remembered why I did this at all while doing something else xD. I will work on this properly tomorrow :)

Reply:
But yes, this is a much better way to put the filename. 👍🏻